tests/simulations/ringtest/create_data.py

```diff
@@ -0,0 +1,158 @@
+#!/bin/env python
+# /// script
+# dependencies = ['h5py', 'libsonata', 'numpy']
+# ///
+# the above allows one to run `uv run create_data.py` without a virtualenv
+
+import h5py
+import libsonata
+import numpy as np
+
+
+def make_nodes():
```

Comment:
Are there the same data for NodeA and NodeB ? If so, can we use a common function?

Reply:
for now, it seems they are, but I'd prefer if they weren't completely alike, so that we know what is contributing to what.
At some point I would like to make helper functions to create all the pieces in a more declarative way, but that will come later, once we have a couple examples of data being created.